src/PostProcess/RenderPipeline/Pipelines/babylon.defaultRenderingPipeline.ts

```diff
@@ -367,6 +367,7 @@
         constructor(name: string = "", hdr: boolean = true, scene: Scene = BABYLON.Engine.LastCreatedScene!, cameras?: Camera[], automaticBuild = true) {
             super(scene.getEngine(), name);
             this._cameras = cameras ||  scene.cameras;
+            this._cameras = this._cameras.slice();

```

Comment:
weird construct :)
we do not want to slice the scene.cameras but just the ones in params